docs/src/upgrade/release-4.x.x.asciidoc

```diff
@@ -106,6 +106,19 @@ that is sent to the endpoint. There is a default interceptor added to every `Clu
 interceptor is responsible for serializing the request body is which what the server normally expects. This is intended
 to be an advanced customization technique that should only be used when needed.
 
+==== Addition of Python interceptor
+
+HTTP interceptors have been added to `gremlin-python` to enable capability similar to that of Java GLV. These
+interceptors can be passed into either a `DriverRemoteConnection` or a `Client` using the interceptors parameter. An
+interceptor is a `Callable` that accepts one argument which is the HTTP request (dictionary containing header, payload
+and auth) or a list/tuple of these functions. The interceptors will run after the request serializer has run but before
+any auth functions run so the HTTP request may still get modified after your interceptors are run. In situations where
+you don't want the payload to be serialized, the `message_serializer` has been split into a `request_serializer` and a
+`response_serializer`. Simply set the `request_serializer` to `None` and this will prevent the `RequestMessage` from
```

Comment:
What happens if they set they mix up the request and response serializers ie graphSON vs graph binary? Is that supported?

Reply:
This is supported. These effectively set the `content-type` and `accept` headers which TinkerPop 4 HTTP servers should handle.